extension/opampextension/README.md

```diff
@@ -43,6 +43,30 @@ extensions:
         endpoint: wss://127.0.0.1:4320/v1/opamp
 ```
 
+## Custom Messages
+
+Other components may use a configured OpAMP extension to send and receive custom messages to and from an OpAMP server. Components may use the provided `components.Host` from the Start method in order to get a handle to the registry:
+
+```go
+func Start(_ context.Context, host component.Host) error {
+	ext, ok := host.GetExtensions()[opampExtensionID]
+	if !ok {
+		return fmt.Errorf("opamp extension %q does not exist", opampExtensionID)
+	}
+
+	registry, ok := ext.(opampextension.CustomCapabilityRegistry)
+	if !ok {
+		return fmt.Errorf("extension %q is not an custom message registry", opampExtensionID)
+	}
+
+  // You can now use registry.Register to register a custom capability
```

Comment:
Could you fix the indentation here? It looks like it was indented with spaces instead of tabs.

Reply:
Should be replaced with tabs now